### ISSUE TITLE
v0.13.2 breaking?

### ISSUE THREAD

**Issue by briochemc** [JuliaMath/Interpolations.jl#420]
It's not a big deal, but I think 

https://github.com/JuliaMath/Interpolations.jl/commit/318ebc88ca1fc084754f3c741266537f901a3310

means that things that used to work now error, so it's a breaking change, and the MINOR version number should have been incremented according to semver, right?

**Comment by mkitti** [comment]
As per #395 repeated knots did not actually work before. It produced `NaN`s. As far as I can tell, it was never intended nor documented for repeated knots to work. My perspective is that the ability to accept repeated knots was a bug and that commit was a bug fix.

Strictly speaking, [according to semver](https://semver.org/#spec-item-4), we are still pre-1.0.0 here:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

I am sorry that your code broke, but I encourage you to review why there are repeated knots and what you intended the result to be.

**Comment by briochemc** [comment]
Sorry I meant pkg's versioning: https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0 which is different to semver specifically for this scenario.

> As per #395 repeated knots did not actually work before.

I understand it did not "work" in the sense that it gave `NaN`s but it did not throw. 

Anyway, it's not important! I fixed it on my side already but I just wanted to point this out here in case someone looks for this as well... And I just realized in the issue you linked that I'm not the first one to notice!
